yarn.lock

```diff
@@ -12273,6 +12273,11 @@ typescript@^5:
   resolved "https://registry.npmjs.org/typescript/-/typescript-5.3.3.tgz"
   integrity sha512-pXWcraxM0uxAS+tN0AG/BF2TyqmHO014Z070UsJ+pFvYuRSq8KH8DmWpnbXe0pEPDHXZV3FcAbJkijJ5oNEnWw==
 
+typescript@^5.4.5:
```

Comment:
Note: You're adding another typescript version. You should be using the same one used by the other modules.

Reply:
In fact this is probably a good moment to pick the same version across the docs and ui module. I would go for `^5.4.5`